onnx/optimizer/optimize.h

```diff
@@ -32,9 +32,10 @@ struct Optimizer {
     passes[sp->name] = std::move(sp);
   }
 
-  std::unique_ptr<onnx::ModelProto> optimize(std::unique_ptr<onnx::ModelProto> mp_in, std::vector<std::string>& names) {
-
-    std::shared_ptr<onnx::Graph> g(onnx::ImportModelProto(*mp_in));
```

Comment:
Actually, if we stick to unique_ptr, we can release the mp_in right after this conversion.

Just call mp_in.reset()

Reply:
While with the new interface that uses const reference, we don't even have a copy from the beginning.